src/helpers.js

```diff
@@ -145,5 +145,9 @@ var deprecate = Marionette.deprecate = function(message, test) {
   }
 };
 
-deprecate._warn = typeof console !== 'undefined' && (console.warn || console.log) || function() {};
+deprecate._console = typeof console !== 'undefined' ? console : {};
+deprecate._console.warn = deprecate._console.warn || deprecate._console.log || function() {};
```

Comment:
Because `deprecate._warn` is called on `deprecate`, and you can't test that `console` is bound.